mautic/mautic
# Add preference_center.tooltip to landing page PageType

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Required: )
#### Description:

Add preference_center.tooltip to landing page PageType

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Create or edit a landing page.
2. Notice the lack of ❔ and tooltip.
![image](https://user-images.githubusercontent.com/4347038/66102819-00866000-e5f7-11e9-8998-b85d92eaa5e9.png)

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create or edit a landing page.
3. See the ❔ icon and tooltip for the preference center `yes/no` button.
![image](https://user-images.githubusercontent.com/4347038/66102963-78ed2100-e5f7-11e9-9aaf-2775fe0bf747.png)
